### PR TITLE
Update events plugin to latest version

### PR DESCRIPTION
**Pull request by sangeetha5491**
Use aio-cli-plugin-events version 3.0.1

## Description

Updates the cli  to use the latest plugin for events

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "version": "9.1.1",
   "author": "Adobe Inc.",
   "bin": {
-    "aio": "./bin/run"
```
Comment (shazron) [inline package.json:7]: I'm sure this won't affect things, but I wouldn't take the risk of updating this here. Should be a separate issue.

**Review by shazron — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (sangeetha5491) [reply package.json:7]: it was updated automatically when i ran npm i. Will revert it.

Comment (sangeetha5491) [reply package.json]: reverted

**Review by shazron — APPROVED** [review]

**Review by moritzraho — COMMENTED** [review]
lgtm

**Pull request merged**